config/default/UserLanguageService.conf.php

```diff
@@ -4,4 +4,5 @@
 
 return new UserLanguageService([
     UserLanguageService::OPTION_LOCK_DATA_LANGUAGE => true,
+    UserLanguageService::OPTION_DEFAULT_LANGUAGE => 'nb-NO',
```

Comment:
- [ ] Please revert this change. This is generis and should not be aware of this. 

```suggestion
```

Reply:
:white_check_mark:  Already reverted: [UserLanguageService.conf.php](https://github.com/oat-sa/generis/blob/feature/ADF-842-configurable-user-default-language/config/default/UserLanguageService.conf.php)